components/ai/code/ai-bbe/why-is-ballerina-the-way-you-should-write-ai-applications.md

```diff
@@ -13,11 +13,14 @@ audio:CreateTranslationRequest translationsReq = {
 
 // Translates the audio file to text (English)
 audio:Client openAIAudio = check new ({auth: {token: openAIKey}});
-audio:CreateTranscriptionResponse transcriptionRes = check openAIAudio->/audio/translations.post(translationsReq);
+audio:CreateTranscriptionResponse transcriptionRes = 
+    check openAIAudio->/audio/translations.post(translationsReq);
 io:println("Audio text in English: ", transcriptionRes.text);
 
-// Creates a request to translate the text from English to another language
-string prmt = string `Translate the following text from English to ${TRANSLATINGLANGUAGE} : ${transcriptionRes.text}`;
+// Creates a request to translate the text from English to other language
+string prmt = string `Translate the following text from English to 
+    ${TRANSLATINGLANGUAGE} : ${transcriptionRes.text}`;
```

Comment:
If we break line here in a string template it will add line break and a tab to the output text. So instead can you break after `${` as follows.

```ballerina
string prmt2 = string `Translate the following text from English to ${
        TRANSLATINGLANGUAGE} : ${transcriptionRes}`;
```